packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -294,7 +292,7 @@ describe("wrangler", () => {
 				await runWrangler("kv namespace list");
 
 				expect(JSON.parse(std.out)).toEqual(kvNamespaces);
-				expect(requests.count).toEqual(6);
+				expect(requests.count).toBeGreaterThan(1);
```

Comment:
Are we sure this change is correct? With this change we're not checking the pagination anymore, are we?
if you want to be more generic here instead of using `toEqual`, what about adding a `toBeLessThan` check as well?

e.g.
```ts
expect(requests.count).toBeGreaterThan(1);
expect(requests.count).toBeLessThan(100);
```

Reply:
PS: I also misread the test... I thought that it was testing the number of kvs... 🤦 yeah I agree with you then that resting the number of requests is just testing an implementation details 😕